submissions/Harnytskyi/html_css_popup/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
```

Comment:
Also -- required meta tag for support IE:
`<meta http-equiv="X-UA-Compatible" content="ie=edge">`